bot_modules/irc_kudos.js

```diff
@@ -74,6 +74,24 @@ module.exports = {
 		return `${what} has no <3`;
 	},
 
+	random: words => {
+		let kudos = ram.get('kudos');
+		let keys = Object.keys(kudos);
+		let randomSelectKey = keys[keys.length * Math.random() << 0];
+		let randomSelectAmount = kudos[randomSelectKey];
+		return `${randomSelectKey} has ${randomSelectAmount} kudos`;
+	},
+
+	amount: words => {
+		words.shift();
+		let count = 0;
+		let amount = parseInt(words);
+		if (isNaN(amount)) return 'yer numb0r is broken!!! D=';
```

Comment:
sorry to nitpick but "broken" should be "borked"

this looks great

Reply:
you're absolutely right and i apologize for this criminal spelling mistake. it is fixed now!

Reply:
i didn't think we'd survive!! ;^__^